easybuild/easyconfigs/l/libobjcryst/libobjcryst-2017.2.2_sconscript_local.patch

```diff
@@ -0,0 +1,20 @@
+--- libobjcryst-2017.2.3/sconscript.local	1970-01-01 01:00:00.000000000 +0100
++++ libobjcryst-2017.2.3/sconscript.local	2020-09-22 15:33:59.098143895 +0200
```

Comment:
If the patch was created using version `2017.2.3`, the patch file should be renamed to reflect that?